vyper/exceptions.py

```diff
@@ -235,11 +235,22 @@ def __init__(self, message=""):
         self.message = message
 
     def __str__(self):
-        return f"{self.message} Please create an issue."
+        return (
+            f"{self.message}\n\nThis is an unhandled internal compiler error. "
+            "Please create an issue on Github to notify the developers."
```

Comment:
Maybe add a link to do this?
https://github.com/vyperlang/vyper/issues/new?template=bug.md